assets/css/src/components/_premiumPage.scss

```diff
@@ -10,6 +10,7 @@
   }
 
   .button.mailpoet-button-bigger {
+    line-height: normal;
```

Comment:
To me the buttons' text looks better vertically aligned without this line.

Reply:
I will take a look. I am using the new WP 5.3 with the new theme Twenty Twenty. It was broken in this theme, that is why I had to add the line.  But with this line it is broken in other themes.